lib/internal/worker.js

```diff
@@ -223,6 +227,12 @@ class Worker extends EventEmitter {
         null,
       hasStdin: !!options.stdin
     }, transferList);
+    // Use this to cache the Worker's loopStart value once available.
+    this[kLoopStartTime] = -1;
```

Comment:
Put the -1 in a const?

Reply:
@benjamingr Sorry, I don't follow what you mean.